### PR TITLE
HPCC-20803 Document Sync up of SSH Keys

### PR DESCRIPTION
**Pull request by g-pan**
Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

@JamesDeFabia please review
@cloLN optionally, please review

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Testing:

http://10.240.32.243/view/Docs%207.0/job/AnotherDocBuild7/

### CODE REVIEW

File: docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml [hunk 1]
```diff
@@ -1161,6 +1161,29 @@ sudo cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/environment.
             Scripts</link>) for more details.</para>
           </listitem>
         </orderedlist>
+
+        <sect3 id="Addl_SSH_KeyInfo">
+          <title>Additional SSH Key Information</title>
+
+          <para>On a multi-node HPCC system, it is quite important that the
+          certificates and SSH keys all match across all nodes in order for
+          the system to work properly. If you used the
```
Comment (JamesDeFabia) [inline docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml]: it is quite important that the certificates and SSH keys all match across all nodes in order for the system to work properly.
s/b
certificates and SSH keys **must** match across all nodes in order for the system to work properly.

File: docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml [hunk 2]
```diff
@@ -1161,6 +1161,29 @@ sudo cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/environment.
...
+          <emphasis>install-cluster.sh</emphasis> script as outlined in the
+          steps above, this could insure that everything is properly in sync.
```
Comment (JamesDeFabia) [inline docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml]: could insure?

File: docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml [hunk 3]
```diff
@@ -1161,6 +1161,29 @@ sudo cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/environment.
...
+          However, it is still a good idea to verify that they do all match
+          up. Another way to ensure this is the case is to use the delivered
+          <emphasis>hpcc-push.sh</emphasis> script. For example, assuming you
+          are running on a CentOS 7 cluster, could use the following commands:
```
Comment (JamesDeFabia) [inline docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml]: Does this only work on Centos 7?

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
see comments inline

**Comment by g-pan** [comment]
@JamesDeFabia please review. 
See http://10.240.32.243/view/Docs%207.0/job/AnotherDocBuild7/

File: docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml [hunk 4]
```diff
@@ -1161,6 +1161,28 @@ sudo cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/environment.
             Scripts</link>) for more details.</para>
           </listitem>
         </orderedlist>
+
+        <sect3 id="Addl_SSH_KeyInfo">
+          <title>Additional SSH Key Information</title>
+
+          <para>On a multi-node HPCC system certificates and SSH keys must all
```
Comment (JamesDeFabia) [inline docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml]: needs a comma between:  system, certificates

File: docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml [hunk 5]
```diff
@@ -1161,6 +1161,28 @@ sudo cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/environment.
...
+          match across all nodes in order for the system to work properly. If
```
Comment (JamesDeFabia) [inline docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml]: match across all nodes for the system to work properly.

File: docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml [hunk 6]
```diff
@@ -1161,6 +1161,28 @@ sudo cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/environment.
...
+          you used the <emphasis>install-cluster.sh</emphasis> script as
+          outlined in the steps above, this would insure that everything is
+          properly in sync. However, it is still a good idea to verify that
+          they do all match up. Another way to ensure this is the case is to
```
Comment (JamesDeFabia) [inline docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml]: Another way to ensure this is the case is to
    sb
Another way to ensure this is to

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
A couple of grammar/style changes noted inline

**Comment by g-pan** [comment]
updated, and rebased. @JamesDeFabia please review

**Review by JamesDeFabia — APPROVED** [review]
good to merge

**Comment by richardkchapman** [comment]
@cloLN Please review

**Comment by g-pan** [comment]
Updated and Rebased.

**Comment by cloLN** [comment]
Looks good.

**Pull request merged**